SignalMessaging/categories/UIFont+OWS.h

```diff
@@ -28,11 +26,20 @@ NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - Dynamic Type
 
-@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeBodyFont;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeTitle1Font;
 @property (class, readonly, nonatomic) UIFont *ows_dynamicTypeTitle2Font;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeTitle3Font;
 @property (class, readonly, nonatomic) UIFont *ows_dynamicTypeHeadlineFont;
-@property (class, readonly, nonatomic) UIFont *ows_infoMessageFont;
-@property (class, readonly, nonatomic) UIFont *ows_footnoteFont;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeSubheadlineFont;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeBodyFont;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeFootnoteFont;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeCaption1Font;
+@property (class, readonly, nonatomic) UIFont *ows_dynamicTypeCaption2Font;
```

Comment:
I'm not sure we'll need all of these, but we now have the canonical list, in correct order.